### PR TITLE
Update numbers in the Fast Section

### PR DESCRIPTION
**Pull request by Hywan**

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -354,28 +354,28 @@ suite, which are known to be important and intensive, both in terms of computati
...
-        <td>26,774</td>
-        <td>177,976</td>
+        <td>27,540</td>
+        <td>411,516</td>
     </tr>
     <tr class="table--double-separator">
         <td>Time</td>
-        <td>25s</td>
-        <td>93s</td>
+        <td>61s</td>
```
Comment (jubianchi) [inline features.md:372]: I think we should tell on which PHP version we got those numbers. With PHP7 on my laptop I'm closer to 20s ;)

File: features.md [hunk 2]
```diff
@@ -354,28 +354,28 @@ suite, which are known to be important and intensive, both in terms of computati
...
+        <td>172s</td>
     </tr>
     <tr>
         <td>Memory</td>
-        <td>58Mb</td>
-        <td>50Mb</td>
+        <td>464Mb</td>
```
Comment (jubianchi) [inline features.md:377]: same thing here.

File: features.md (hunk 1 above)
Comment (Hywan) [inline features.md:372]: PHP7 on Travis.

File: features.md (hunk 2 above)
Comment (Hywan) [inline features.md:377]: PHP7 on Travis.

**Pull request merged**